docs/customize/metadata/custom_fields/records.md

```diff
@@ -596,4 +602,4 @@ The previous configuration produces the following result in the deposit form:
 
 and landing page:
 
-![Reusable custom field result (landing page)](../../img/custom_fields_example_landing_page.png)
\ No newline at end of file
+![Reusable custom field result (landing page)](../../img/custom_fields_example_landing_page.png)
```

Comment:
what changed here?

Reply:
Nothing. I have no idea why GitHub flagged it